lib/fastlane/actions/slack.rb

```diff
@@ -27,7 +27,9 @@ def self.run(options)
 
         notifier = Slack::Notifier.new(options[:slack_url])
 
-        notifier.username = 'fastlane'
+        notifier.username = options[:override_slack_username_and_icon] ? options[:username] : nil
+        icon_url = options[:override_slack_username_and_icon] ? options[:icon_url] : nil
```

Comment:
can we default these values to `'fastlane'` and `'https://s3-eu-west-1.amazonaws.com/fastlane.tools/fastlane.png'` instead of nil?